docs/plugin-syntax-typescript.md

```diff
@@ -38,6 +38,22 @@ require("@babel/core").transformSync("code", {
 
 ## Options
 
+### `disallowAmbiguousJSXLike`
+
+`boolean`, defaults to `false`
+
+Added in: `v7.16.0`
+
+Even when JSX parsing is not enabled, this option disallows using syntax that would be ambiguous with JSX (`<X> y` type assertions and `<X>() => {}` type arguments). It matches the `tsc` behavior when parsing `.mts` and `.mjs` files.
+
+### `dts`
+
+`boolean`, defaults to `false`
+
+Added in: `v7.14.0`
```

Comment:
```suggestion
Added in: `v7.20.0`
```
Although the `dts` parser option has been supported since 7.14.0, this option is not passed by the syntax / transform plugin until 7.20.0, so we should put 7.20.0 here.